### DISCUSSION TITLE
How to kill k3s server?

### DISCUSSION THREAD

**Discussion by sshnuke3** [k3s-io/k3s#7340] [General]
I didn't install k3s as a service

I just downloaded k3s elf file and use k3s server & to start it

So how can I stop them in one command?

**Comment by brandond** [comment]
`fg` and then control-C? Or find the pid in the `ps` output and kill it? `pkill k3s` would work if you have that available. Basically the same way you would any other background process.

**Comment by Ranburu** [comment]
> `fg` and then control-C? Or find the pid in the `ps` output and kill it? `pkill k3s` would work if you have that available. Basically the same way you would any other background process.

K3s process just restore himself after killing the process, and what shoud I do to avoid it?

**Comment by brandond** [comment]
The k3s process will not restart itself if you've run it directly from a shell as `k3s server &`. Are you perhaps killing one of the k3s child processes instead?

**Comment by caroline-suse-rancher** [comment]
Converting this one to a discussion
